src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1802,6 +1802,16 @@ open class StateManagerAdaptor(
         callback(false, V4TransactionErrors.error(null, "Not implemented"), null)
     }
 
+    internal open fun triggerCompliance(action: ComplianceAction, callback: TransactionCallback) {
```

Comment:
Similar to your other PR, what is the purpose of the `callback` here? We're not doing anything asynchronous here.